src/libraries/System.Globalization/tests/CultureInfo/CultureInfoCurrentCulture.cs

```diff
@@ -33,6 +33,14 @@ public void CurrentCulture()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.OSX | TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS)]
```

Comment:
Is this not going to pass on windows and Linux too?

Reply:
It is possible to fail on Linux and Windows. On Windows it is unlikely the users will set the user locale to Invariant but it is possible. For Linux, some distros set the locale to "C" which we replace it with Invariant. That means it is likely to fail on Linux.